apps/backend-functions/src/event.ts

```diff
@@ -30,6 +33,13 @@ export async function onEventDelete(snap: BlockframesSnapshot<Event<EventMeta>>)
   return batch.commit();
 }
 
+export async function onEventCreate(snap: BlockframesSnapshot<Event<EventMeta>>) {
+  const event = snap.data();
+  const from = getMailSender('festival');
+  
+  await sendMail(sendEventCreatedEmail('festival', event), from);
```

Comment:
```suggestion
  const emailData = sendEventCreatedEmail('festival', event);
  await sendMail(emailData, from);
```
I would have used a separate const to prevent function stacking.